### PR TITLE
ImportFromGit functionality for the Pipeline Entity

### PR DESCRIPTION
**Pull request by vtxorxwitty**
Changes to support importFromGit functionality for the Pipeline Entity. 

Tested the terraform changes with these on automation four QA account. We are able to import a pipeline from the given filepath on git and create a Pipeline entity in harness UI.

### CODE REVIEW

File: nextgen/api_pipelines.go [hunk 1]
```diff
@@ -562,33 +809,34 @@ func (a *PipelinesApiService) ListPipelines(ctx context.Context, org string, pro
...
+@return PipelineMoveConfigResponseBody
 */
 
-type PipelinesApiUpdatePipelineOpts struct {
+type PipelinesApiMoveConfigOpts struct {
+    Body optional.Interface
     HarnessAccount optional.String
 }
 
-func (a *PipelinesApiService) UpdatePipeline(ctx context.Context, body PipelineUpdateRequestBody, org string, project string, pipeline string, localVarOptionals *PipelinesApiUpdatePipelineOpts) (PipelineCreateResponseBody, *http.Response, error) {
```
Comment (rathodmeetsatish) [inline nextgen/api_pipelines.go:581]: @rohitk-h @vtxorxwitty is removing this method backward compatible?

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

**Comment by RajBaviskar** [comment]
@vtxorxwitty Please fix the PR title and ad one-liner description as well.

File: nextgen/api_pipelines.go (hunk 1 above)
Comment (rohitk-h) [reply nextgen/api_pipelines.go:581]: The method is not removed, diff is not showing correctly. The method still exists in the new changes at this location: https://github.com/harness/harness-openapi-go-client/pull/13/files#diff-3e3990f4e07e7710d994b515851fc4db0db450af860cb0b66d4ca2635766e56fR1085

Comment (rathodmeetsatish) [reply nextgen/api_pipelines.go:581]: ok

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**